site/content/blogs/release-v127.ja.md

```diff
@@ -0,0 +1,367 @@
+---
+title: 'AWS Copilot v1.27: Copilot テンプレートの拡張、追加のルーティングルールのサポート、差分のプレビュー、サイドカーの拡張!'
+twitter_title: 'AWS Copilot v1.27'
+image: 'https://user-images.githubusercontent.com/879348/227655119-e42c6b8b-ff0e-4abe-ad90-89b44813fbd5.png'
+image_alt: 'CDK overrides and diff'
+image_width: '2056'
+image_height: '1096'
+---
+
+# AWS Copilot v1.27: Copilot テンプレートの拡張、追加のルーティングルールのサポート、差分のプレビュー、サイドカーの拡張！
+##### 投稿日: 2023 年 3 月 28 日
+
+AWS Copilot コアチームは Copilot v1.27 のリリースを発表します 🚀。
+私たちの公開[コミュニティチャット](https://gitter.im/aws/copilot-cli) は増え続けて、400 人以上がオンラインで、[GitHub](http://github.com/aws/copilot-cli/) には 2,700 以上のスターを獲得しています。
+AWS Copilot へご支援、ご支持いただいている皆様お一人お一人に感謝をいたします。
+
+Copilot v1.27 は幾つかの新機能と改善点がある大きなリリースです。
+
+- **Copilot テンプレートの拡張**: Copilot が生成した AWS CloudFormation テンプレートの任意のプロパティをカスタマイズできる様になりました。
+AWS Cloud Development Kit (CDK) または YAML パッチオーバライドを使用します。[詳細セクションはこちらをご覧ください](#extend-copilot-generated-aws-cloudformation-templates)。
+- **複数のリスナーとリスナールールが有効に**: [Application Load Balancer](../docs/manifest/lb-web-service.ja.md#http)にはホストベースまたはパスのリスナーのルールを、
+[Network Load Balancer](../docs/manifest/lb-web-service.ja.md#nlb)に異なるポートやプロトコルのリスナーを複数定義できます。
+[詳細セクションはこちらをご覧ください](#enable-multiple-listeners-and-listener-rules-for-load-balancers).
+- **CloudFormation テンプレート変更のプレビュー**: `copilot [noun] package` または `copilot [noun] deploy` コマンドに `--diff` フラグを付けて実行すると、
+最後にデプロイされた CloudFormation テンプレートとローカルの変更における差分を表示できる様になりました。[詳細セクションはこちらをご覧ください](#preview-aws-cloudformation-template-changes)。
+- **サイドカーのコンテナイメージのビルドとプッシュ**: ローカルの Dockerfile からサイドカーコンテナをビルドしてプッシュするための `image.build` をサポートしました。[詳細セクションはこちらをご覧ください](#build-and-push-container-images-for-sidecar-containers)。
+- **サイドカーに対する Environmment ファイルのサポート**: サイドカーコンテナ用のローカルな `.env` ファイルをプッシュするための `env_file` をサポートしました。[詳細セクションはこちらをご覧ください](#upload-local-environment-files-for-sidecar-containers)。
+
+??? note "AWS Copilot とは？"
+
+    AWS Copilot CLI は AWS 上でプロダクションレディなコンテナ化されたアプリケーションのビルド、リリース、そして運用のためのツールです。
+    開発のスタートからステージング環境へのプッシュ、本番環境へのリリースまで、Copilot はアプリケーション開発ライフサイクル全体の管理を容易にします。
+    Copilot の基礎となるのは、 AWS CloudFormation です。CloudFormation により、インフラストラクチャを 1 回の操作でコードとしてプロビジョニングできます。
+    Copilot は、さまざまなタイプのマイクロサービスの作成と運用の為に、事前定義された CloudFormation テンプレートと、ユーザーフレンドリーなワークフローを提供します。
+    デプロイメントスクリプトを記述する代わりに、アプリケーションの開発に集中できます。
+
+    より詳細な AWS Copilot の紹介については、[Overview](../docs/concepts/overview.ja.md) を確認してください。
+<a id="extend-copilot-generated-aws-cloudformation-templates"></a>
+## Copilot が生成した AWS CloudFormation テンプレートの拡張
+
+AWS Copilot を利用すると、`copilot init`　コマンドとプロンプトに従うだけで、ビルダーはコンテナ化されたアプリケーションを素早く開始できます。
+その後、開発者は [Manifest](../docs/manifest/overview.ja.md) の infrastructure-as-code ファイルを編集し、デプロイすることで、アプリケーションを拡張できます。
+さらに v1.27 では、 Copilot が生成した Cloudformation テンプレートを `copilot [noun] override` コマンドで拡張できるようになり、インフラストラクチャーを完全にカスタマイズできる様になりました。
```

Comment:
Sorry for nitpicking: Cloudformation to CloudFormation

Reply:
Understood.